cypress/integration/page.spec.js

```diff
@@ -421,7 +450,7 @@
     describe('if posts are loaded successfully', () => {
       beforeEach(() => {
         page.mockUsers();
-        page.mockUser1Posts()
+        page.mockUser1Posts();
```

Comment:
There is a missing semicolon at the end of the `cy.intercept` statement. Ensure that each statement is properly terminated to avoid syntax errors.